Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -428,6 +428,7 @@ - (void)toxAV:(OCTToxAV *)toxAV callStateChanged:(OCTToxAVCallState)state friend
 
         if ((self.audioEngine.friendNumber == friendNumber) && [self.audioEngine isAudioRunning:nil]) {
             [self.audioEngine stopAudioFlow:nil];
+            [self.videoEngine stopSendingVideo];
```

Comment:
Currently not but I guess it would be better to put the stop video engine in a separate line to guard against it in the future.

`if ((self.videoEngine.friendNumber == friendNumber) && [self. self.videoEngine isRunning:nil]) `